packages/cz-lerna-changelog/lib/index.js

```diff
@@ -63,23 +63,28 @@ function makePrompter() {
 
     cz.prompt(autocompleteQuestions(questions))
       .then((answers) => {
-        const { body, footer, ...rest } = answers
+        const { body, testplan, visualChange, footer, ...rest } = answers
+
+        const testplanTxt = testplan ? `\nTEST PLAN:\n${testplan}\n\n` : ''
 
-        const testplan = answers.testplan
-          ? `\nTEST PLAN:\n${answers.testplan}\n\n`
-          : ''
         const issues = footer ? `\n\nCloses: ${footer}\n\n` : ''
 
+        const visualChangeTxt = visualChange
+          ? `\n\nVisual change: ${visualChange}\n\n`
```

Comment:
So far every, every major topic (e.g.: TEST PLAN) is with all capital letters, shouldn't we keep this pattern?
```suggestion
          ? `\n\nVISUAL CHANGE: ${visualChange}\n\n`
```

Reply:
good idea, I'll change it